1.20.0/schemas/dcr/ClientRegistration.json

```diff
@@ -0,0 +1,46 @@
+{
```

Comment:
There used to be a ClientRegistrationRequest in the 1.19.0 which included the schema part for the jwt string to be decoded, which wasn't included in the dcr folder. It this on purpose?

Reply:
I have added this again.